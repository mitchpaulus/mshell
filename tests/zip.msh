[1 2 3] [4 5 6] (+) zip (str) map uw
[1 2] [3 4 5 6 7 8] (+) zip (str) map uw
[3 4 5 6 7 8] [1 2] (+) zip (str) map uw

["a" "b"] ["c" "d"] (b! a! [] @a append @b append tjoin) zip uw

"2tuple test" wl
["a" "b" "c"] dup len seq (2tuple (str) map) zip tuw
