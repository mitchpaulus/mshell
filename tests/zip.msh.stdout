5
7
9
4
6
4
6
a	c
b	d
2tuple test
a	0
b	1
c	2
